weekly.go

```diff
@@ -150,6 +149,13 @@ func runWeelyCommandFunc(cmd *cobra.Command, args []string) {
 	title := sprint.Name
 	createWeeklyReport(title, body.String())
 
+	pendingIssues := getIssuesForSprintWithFilter(sprint.ID, func(issue *jira.Issue) bool {
```

Comment:
Seems that using JQL is a better solution so that the filter can be performed at server side.

Reply:
Well, I think a simple filter in client side is enough for our case.

Reply:
It would be very nice if you can make your function useful in more cases, for example, https://github.com/siddontang/work-reporter/pull/8